Makefile

```diff
@@ -16,12 +16,11 @@ test:
 doctest:
 	$(PYTHON) -c "import skimage, sys, io; sys.exit(skimage.doctest_verbose())"
 
-coverage:
-ifeq ($(suite),$(BENCHMARK))
+benchmark_coverage:
 	$(PYTESTS) benchmarks --cov=skimage
-else
+
+test_coverage:
```

Comment:
I would keep this one the same. benchmarks are rather experimental for now. but these are just my 2cts. @jni might have different organizational preferences.

If you want you can have

```
coverage: test_coverage
```

(`test_coverage` is a dependency of `coverage`)

Reply:
Sorry, but I'm new to this. When I tried this, it is throwing an error. See

When I tried doing this,
```
coverage: test_coverage
	$(PYTESTS) -o python_functions=test_* skimage --cov=skimage
```
It's throwing errors.

`make: *** No rule to make target `test_coverage', needed by `coverage'.  Stop.`

Is that what I was supposed to do?

Reply:
Yeah, makefiles are confusing. It would simply be a new line with no real other purpose than making a shortcut from `corvarage` -> `test_coverage`

See http://mrbook.org/blog/tutorials/make/ `Using dependencies`

Reply:
Ohh, thanks @hmaarrfk. That was helpful. Done with the changes.

Reply:
Just as an note, this means that you can now run
```
make coverage benchmark_coverage
```
and it will actually test both coverages. With the previous flag approach you had, it would have not been possible

Reply:
It only runs tests and not the benchmark. Idk what's wrong.

Reply:
Actually, tests are throwing some errors. So, the makefile execution stopped after the first command exited with an error. I used the `--keep-going` flag and it works fine. Thanks @hmaarrfk.